pkg/reconciliation/reconcile_fql.go

```diff
@@ -1,98 +1,53 @@
 package reconciliation
 
 import (
-	"encoding/json"
 	"errors"
-	"fmt"
-	"strconv"
-	"strings"
 
 	"github.com/k8ssandra/cass-operator/pkg/httphelper"
 	"github.com/k8ssandra/cass-operator/pkg/internal/result"
 )
 
-// parseFQLFromConfig parses the DC config field to determine whether FQL should be enabled based on the presence of full_query_logging_options within the cassandra-yaml field.
-// To ease integration into the reconciliation process, it returns (shouldFQLBeEnabled, serverMajorVersion, ReconcileResult) where the ReconcileResult may be result.Error() or result.Continue().
-func parseFQLFromConfig(rc *ReconciliationContext) (bool, int64, result.ReconcileResult) {
+// CheckFullQueryLogging sets FQL enabled or disabled based on the `enableFQL` parameter, and takes serverMajorVersion for additional validation.
```

Comment:
This isn't quite accurate since it doesn't take an `enableFQL` parameter or serverMajorVersion.